### PR TITLE
Style generated report

### PR DESCRIPTION
**Pull request by lmichalevich**
## Status

- [x] Ready
- [ ] In Progress

## Related Issues

https://github.com/demisto/etc/issues/38832 (Partial styling)

Fixes https://github.com/demisto/etc/issues/40663 (is included in first issue)

Fixes https://github.com/demisto/etc/issues/40665 (is included in first issue)

## Related PRs

https://code.pan.run/xsoar/server/-/merge_requests/16211

https://code.pan.run/xsoar/web-client/-/merge_requests/6618

## Description

Style generated report (some basic styling - the first issue is too big to be in a single PR)

Including:
- fix bug when the team members avatars were too bug
- fix bug that the properties keys in a card section broke (they had unnecessary width: 30% on them)
- fix bug that some sections had missing borders
- update fonts / paddings
- update no items font icon
- remove top right triangle
- for incidents info -> update the logo position (not affecting the logos in the generated reports that are on every page)

### Before
Screen Shot 2021-08-29 at 14 03 57

### After 
Screen Shot 2021-08-29 at 19 20 28

### CODE REVIEW

File: src/components/Sections/ItemsSection.less [hunk 1]
```diff
@@ -4,30 +4,45 @@
...
+        padding-bottom: 9px;
+        word-break: break-word;
+      }
+    }
+
     &.row {
       min-height: 24px;
       flex-direction: row;
       justify-content: space-between;
+      align-items: center;
```
Comment (yanyo87) [inline src/components/Sections/ItemsSection.less:32]: i'm not sure about this

File: src/components/Sections/ItemsSection.less [hunk 2]
```diff
@@ -4,30 +4,45 @@
...
       border-bottom: 1px solid rgba(137, 165, 193, 0.2);
+      padding: 10px 0;
```
Comment (yanyo87) [inline src/components/Sections/ItemsSection.less:34]: i'm not sure about this

**Review by yanyo87 — APPROVED** [review]
looks very good ! 

@Bars92 please have a look as well

File: src/components/Sections/ItemsSection.less (hunk 1 above)
Comment (lmichalevich) [reply src/components/Sections/ItemsSection.less]: I removed it

**Pull request merged**